android/app/src/main/java/com/awesometsproject/MainApplication.java

```diff
@@ -4,6 +4,7 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.oblador.vectoricons.VectorIconsPackage;
```

Comment:
<img width="398" alt="Screenshot 2021-12-27 at 00 38 40" src="https://user-images.githubusercontent.com/58725560/147420655-0b98a646-91ad-46c0-9818-0508502deb6e.png">

I don't see any of icon which means "react-native-vector-icons" package is not connected. properly.

I would suggest using of recommended way to connect:
https://github.com/oblador/react-native-vector-icons#option-with-gradle-recommended